addons/mail/static/src/messaging/component/follower_list_menu/follower_list_menu.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.messaging.component.FollowerListMenu" owl="1">
+        <div class="o_FollowerListMenu">
+            <t t-if="thread.isCurrentPartnerFollowing">
+                <button class="o_FollowerListMenu_unfollowButton btn btn-link" t-att-disabled="props.isDisabled" t-on-click="_onClickUnfollow" t-on-mouseenter="_onMouseEnterUnfollow" t-on-mouseleave="_onMouseLeaveUnfollow" t-att-class="{ 'o-following': !state.hasUnfollowHover, 'o-unfollow': state.hasUnfollowHover }">
+                    <t t-if="state.hasUnfollowHover">
+                        <i class="fa fa-times"/> Unfollow
+                    </t>
+                    <t t-else="">
+                        <i class="fa fa-check"/> Following
+                    </t>
+                </button>
+            </t>
+            <t t-else="">
+                <button class="o_FollowerListMenu_followButton btn btn-link" t-att-disabled="props.isDisabled" t-on-click="_onClickFollow">
+                    Follow
+                </button>
+            </t>
```

Comment:
I feel like this code (about the (un)follow button) should not be on this component, it has nothing to do with the follower list or the dropdown menu, it's another element of the topbar.

Reply:
Also this toggle button keeps making no sense to me.

Unselected state text:
- Follow (standard)
- Follow (hover)

Select state text:
- Following (standard)
- Unfollow (hover)

The standard state text is not consistent between the two cases. One time it tells the current state, one time it tells the action.
The hover state text is consistent.

Reply:
Initially the component was named "FollowersMenu" and I am potentially a follower so it made sense for me.

The toggle button is bad in my opinion too but I'm not sure of what the PO's want for that place so I would not make a change on this without talking about it with Alexandra

Reply:
I'm not really re-questioning the name here, but those are 2 different chatter topbar buttons, and if we deem necessary to make a component for them instead of directly putting them on chatter topbar (which is think is good), then I feel we should go all the way and make them two different components.

One being the FollowerListMenu and the other just being FollowButton for example.
They have almost nothing in common, all handlers are different, different css, etc. It would be easier to maintain and make cleaner code to split them.

OK to not change the toggle mechanisms itself as part of this PR.